subsys/bluetooth/controller/hci_driver.c

```diff
@@ -23,6 +23,12 @@
 #define BLE_CONTROLLER_IRQ_PRIO_LOW  4
 #define BLE_CONTROLLER_IRQ_PRIO_HIGH 0
 
+#if IS_ENABLED(CONFIG_SOC_SERIES_NRF52X)
+	#define BLE_CONTROLLER_PROCESS_IRQn SWI5_IRQn
+#elif IS_ENABLED(CONFIG_SOC_SERIES_NRF53X)
+	#define BLE_CONTROLLER_PROCESS_IRQn EGU0_IRQn
+#endif
```

Comment:
51?

Should we error-out on unrecognized platforms?

Reply:
Ah, this the NRFXLIB_BT_LL ? Nevermind, sorry.

Reply:
You cannot select the controller for 51.